.circleci/config.yml

```diff
@@ -56,10 +56,12 @@ jobs:
       - run: |
           nix-env -iA cachix -f https://cachix.org/api/v1/install
           cachix use vulcanclimatemodeling
-          nix-build -j 4 -A fv3 | cachix push vulcanclimatemodeling
+      - run: |
+          nix-build -j 4 -A fms | cachix push vulcanclimatemodeling
+          nix-shell --run "cp -f nix/fv3/configure.fv3 FV3/conf && make -j 4 -C FV3"
       - run:
           name: Test
-          command: nix-shell tests.nix --run tox
+          command: nix-shell --run "pytest --native tests/pytest"
```

Comment:
why is the --native flag used still under the 'nix + docker' category? I'm not sure what docker means in this context

Reply:
this flags selects only these tests that can be run assuming `FV3/fv3.exe` exists. The build command above makes this executable.

Are you referring to the fact that the circle CI is running this in a `docker` container? That isn't super relevant, just a means of getting an environment with nix on Circle CI.